tests/e2e/test_api.py

```diff
@@ -68,11 +163,18 @@ def test_invalid_question():
     }
     assert response.json() == expected_json
 
+    # new_llm_calls = get_model_provider_counter_value(client, "llm_calls_total",
+    #                                                 "ibm/granite-13b-chat-v2",
+    #                                                 "bam")
+    # check_counter_increases(endpoint, old_llm_calls, new_llm_calls)
```

Comment:
What is the deal with that commented code?

Reply:
yup, will be added later. now I removed them